travis/docker/requirements.txt

```diff
@@ -1,17 +1,18 @@
-sphinxcontrib-napoleon
-sphinxcontrib-autoprogram==0.1.4
+capstone
```

Comment:
I think the file rename is misleading.  Does it at least work on CI?

Reply:
It worked in the CI after I edited .travis.yml. I was copying the requirements.txt into the Docker container to force any changes I made to be introduced into the container. The original design clones from the main pwntools repo, so it wouldn't introduce new changes. Docker restricts copying files from a parent directory into the container, so I moved the location to make it work. I thought I needed the coverage module, but I think it is accounted for someplace else. 

So, overall this is extraneous, and I am moving the file back to the original location. It may be worth keeping the list in alphabetical order. The latest version of Sphinx appears to work now with the build.